### ISSUE TITLE
Black Screen on Boot After Week 48.5, 2023 Updates

### ISSUE THREAD

**Issue by davidhfo** [getsolus/packages#991]
### Summary

After updates from Week 48.5, 2023 ­— updates which completed with no errors — booting on the latest kernel now goes to a black screen.

Booting to 6.3 works ok.

### Steps to reproduce

Update to latest Solus version with similar setup, I suppose! :)

### Expected result

To see log-in screen and be able to log in as usual.

### Actual result

No log-in screen appeared. Only black screen. Could not switch to other TTYs.

### Environment

- [X] Is system up to date?

### Repo

Shannon (stable)

### Desktop Environment

Plasma

### System details

Operating System: Solus 4.4
KDE Plasma Version: 5.27.8
KDE Frameworks Version: 5.110.0
Qt Version: 5.15.11
Kernel Version: 6.3.8-240.current (64-bit)
Graphics Platform: X11
Processors: 4 × Intel® Core™ i5-7300HQ CPU @ 2.50GHz
Memory: 7.7 GiB of RAM
Graphics Processor: Mesa Intel® HD Graphics 630
Manufacturer: LENOVO
Product Name: 80WK
System Version: Lenovo Y520-15IKBN

### Other comments

Running Solus on an NVMe drive in an SSD enclosure connected via USB.

**Comment by TraceyC77** [comment]
@davidhfo - Does this still happen with an updated system? There have been some kernel updates that may have resolved this issue.

**Comment by davidhfo** [comment]
Hi, @TraceyC77. Yes, the system is up to date, and I still have to use the old kernel for it. Otherwise it just shows the black screen.

**Comment by davidhfo** [comment]
I don't know if this makes any difference, but I've noticed that it's more likely to show a splash screen and just hang there, than it is to just have a black screen. In either case, it doesn't progress anywhere past that that I can see, even after waiting for several minutes.

I try periodically after updates to the kernel, to boot into the new one, but it's still the same.

**Comment by ReillyBrogan** [comment]
So this is still an issue with 6.16.16-273?

**Comment by ReillyBrogan** [comment]
Would you be willing to run unstable for a while and hop on Matrix? This is likely going to take many kernel builds to figure out and it's going to be much faster if we can coordinate that over chat.

**Comment by ermo** [comment]
@davidhfo Is this still an issue? If not, please close. If it is, mention @ReillyBrogan .

**Comment by davidhfo** [comment]
Yes, it remains an issue. @ReillyBrogan has done some troubleshooting on it for me.

I was considering wiping and reinstalling to see if that helps. But if it that doesn't work and then I still get the black screen, would there be any way to install a specific and different kernel?

**Comment by davidhfo** [comment]
Periodically, after an update, I will test out the current kernel to see if it works. I had been doing it less often because it just wasn't working.

But I tested it after the week 36 updates, and then I could use the current kernel!

Tried several restarts and it was fine each time. Tested again after week 37, and also fine.

So I don't know what the problem was causing it, or how it was finally fixed, but I'm glad it has been.

This can be closed. Thank you!
